grouper/api/routes.py

```diff
@@ -14,6 +15,7 @@
 
     (r"/users", Users),
     (r"/users/{}".format(NAME_VALIDATION), Users),
+    (r"/bulkusers", BulkUsers),
```

Comment:
nit: looking forward it'd be nice to have other bulk/multi endpoints. can we use the `multi` route prefix? i.e. `(r"/multi/users", BulkUsers)`. We should maybe also rename `BulkUsers` to `MultiUsers` to be consistent.